.github/workflows/e2e_tests.yml

```diff
@@ -0,0 +1,281 @@
+name: e2e-tests
+
+on:
+  push:
+    tags:
+      - '*tetrate*'
+
+jobs:
+  create-test-images:
+    strategy:
+      matrix:
+        os: [ ubuntu-latest ]
+    name: create-test-images
+    runs-on: ${{ matrix.os }}
+    env:
+      HUB: hellozee-docker-istio-testing.bintray.io # change
+
+    steps:
+      - name: checkout
+        uses: actions/checkout@v2
+
+      - name: docker-login
+        uses: docker/login-action@v1
+        with:
+          registry: hellozee-docker-istio-testing.bintray.io # change
+          username: ${{ secrets.BINTRAY_USER }}
+          password: ${{ secrets.BINTRAY_API_KEY }}
+
+      - name: build and push images
+        shell: 'script -q -e -c "bash {0}"'
+        run: |
+          export TAG=$(git describe --tags)
+          make docker push
+
+  eks-e2e-test:
+    strategy:
+      matrix:
+        os: [ ubuntu-latest ]
```

Comment:
Instead use eks versions. And pass this to create the eks cluster and test out on different versions of eks